internal/helloworld/helloworld.proto

```diff
@@ -0,0 +1,35 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Why is this needed? Can we use one of OTLP request/responses for testing instead?

Reply:
Good idea, I can do that.

Reply:
Done.